.github/workflows/ci.yml

```diff
@@ -110,12 +93,31 @@ jobs:
         shell: bash
         run: |
           # Prevent setuptools from replacing distutils with its vendored version, as our test suite is not
-          # ready for that.
+          # ready for that. There also seem to be issues with setuptools-provided distutils and pip 22.2
+          # on python < 3.10.
           echo "SETUPTOOLS_USE_DISTUTILS=not-today" >> $GITHUB_ENV
```

Comment:
I found out recently that you can set environment variables job-wide just by popping them at the top like [this](https://github.com/ultrajson/ultrajson/blob/264f60c018fa79fe5cb0eaee7acadf1ad3eeecb4/.github/workflows/deploy.yml#L16-L17),

Reply:
Ohhh, that's much nicer.